lib/util/cache/package/index.ts

```diff
@@ -15,7 +15,7 @@ export function get<T = any>(namespace: string, key: string): Promise<T> {
     return undefined;
   }
   const globalKey = getGlobalKey(namespace, key);
-  if (!memCache.get(globalKey)) {
+  if (memCache.get(globalKey) === undefined) {
```

Comment:
@JamieMagee  I had to slightly modify the caching logic to enable caching of null values

Reply:
See comment above